package.json

```diff
@@ -24,27 +24,27 @@
   },
   "peerDependencies": {
     "eslint": "^8.34.0",
-    "prettier": "^1.19.0 || ^2.0.0",
+    "prettier": "^3.0.0",
     "typescript": "^3.7.0 || ^4.0.0 || ^5.0.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.9.0",
-    "@typescript-eslint/parser": "^6.9.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.29.0",
-    "eslint-plugin-jest": "^27.6.0",
+    "@typescript-eslint/eslint-plugin": "^7.1.1",
+    "@typescript-eslint/parser": "^7.1.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.9.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react": "^7.34.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-unicorn": "^48.0.1",
-    "husky": "^8.0.3",
-    "lint-staged": "^13.2.2"
+    "eslint-plugin-unicorn": "^51.0.1",
```

Comment:
This upgrade has likely introduced a lot of new rules so we should test this thoroughly

Reply:
Agree.  `esling-plugin-unicorn` appears to have a somewhat relaxed idea of when a major release is required:

https://github.com/sindresorhus/eslint-plugin-unicorn/releases

Notable changes:

- [`consistent-destructuring`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/consistent-destructuring.md) removed from `recommended` preset
- [`no-unnecessary-polyfills`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-unnecessary-polyfills.md) new rule - I doubt we'll have any issues with this.

I will test this in a few more services and report back.

Reply:
consistent-destructuring seems nice to have.